content/millennium/r4/other/binary.md

```diff
@@ -42,7 +42,7 @@ _Implementation Notes_
 
 _Implementation Notes_
 
-* The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
+* The `Binary.read` scope and either one of the `DiagnosticReport.read`, the `DocumentReference.read`, or the `Communication.read` scope is required.
```

Comment:
Can I retrieve a DiagnosticReport or DocumentReference with the Communication scope?  No, right?
I think you can flip Communication first 
The `Binary.read` scope and `Communication.read` scope or either the `DiagnosticReport.read` or the `DocumentReference.read` is required.

Reply:
I mean.... This is just a giant or.... which is a list like it's already written. I think you want something different like a list of options (which might not be able to be conveyed in a super long list)

```
Communication.read scope or either the DiagnosticReport.read or

Reply:
I think Evan and I are saying the same thing.  Evan just didn't finish the sentence.

Reply:
I did finish my thoughts. I just copied the part of yours that is just a giant or clause

Reply:
https://github.com/cerner/fhir.cerner.com/pull/607/commits/73cd84784ce49a7b41ed138be9e5294f9ab8c264

Reply:
Im not in favor of the wording. Going to break down the parts I see:

The `Binary.read` scope and `Communication.read` scope or either the `DiagnosticReport.read` or the `DocumentReference.read` is required.

Binary.read` scope and `Communication.read` - this part seems independent of the rest of the sentence. Also seemingly makes it seem like Communication is required all the time.

or either - using this as a phrase is contradictory. Definition: used before the first of two (or occasionally more) alternatives that are being specified (the other being introduced by “or”).
"either I'll accompany you to your room, or I'll wait here"

Binary and Communication or either DiagnosticReport or DocumentReference = Binary and Communication or DiagnosticReport or DocumentReference
Which also means that the only required scope is Binary, and you just need one of the other scopes.

So at the end of this, Im still confused as to the direction this wording is going in.
I would think that just in it's current format you could do something like: 
```
The required scopes are the `Binary.read` scope, as well as one of the following scopes: `Communication.read`, `DiagnosticReport.read`, or `DocumentReference.read`.
```
and understand it better.

Reply:
That's fine

Reply:
https://github.com/cerner/fhir.cerner.com/pull/607/commits/1287fa77649531cdaf02fcc169b1a67612536562#diff-b9330978e62a15a61d6612ae8dc779e3b4680761c2dfc7e9b3778909d2d95cd8L45